SUMMARY.md

```diff
@@ -1,7 +1,8 @@
 # Summary
 
 ### Learn RxJS
-
+- [Internal](/internal/README.md)
```

Comment:
I think creating a new section for `Subjects` is fine. Maybe we just call it `Subjects` instead of internal and include `Subject`, `BehaviorSubject`, `ReplaySubject`, `AsyncSubject` with explanations of each and how they differ. I also think we can move this under the operators in the main TOC. I will try to do a general write up as well of Subjects that can be included as a general overview.

Reply:
sounds good, I'll get it moved tomorrow

Reply:
that's subjects section added

Reply:
I'd personally merge that in without having all subject variants as I think people would benefit from this example to start with. But it's obviously up to you :)

Reply:
Nevermind, I actually had time to add simple examples of subject variants :) can also create readme for subjects if you haven't started already?

Reply:
Whoa, you have been busy! Feel free to start the readme if you want, reviewing the other subjects now. Thanks for all of your help!